### PR TITLE
Fix - TRSV2-431 - Feedback fixes - master

### PR DESCRIPTION
**Pull request by chris-pettinga**

### CODE REVIEW

File: trade_remedies_public/templates/v2/feedback/collect_feedback.html [hunk 1]
```diff
@@ -162,7 +172,8 @@ <h1 class="govuk-heading-xl">
                     </div>
                 </div>
             </div>
-            <p>We use feedback to help us improve the service and cannot respond to any requests for support you make on this form.</p>
+            <p>We use feedback to help us improve the service and cannot respond to any requests for support you make on
+                this form.</p>
```
Comment (yusufsheiqh) [inline trade_remedies_public/templates/v2/feedback/collect_feedback.html:176]: Do we need to run `djhtml` on this branch? This seems to have separated out on to two lines.

File: trade_remedies_public/templates/v2/feedback/collect_feedback.html [hunk 2]
```diff
@@ -173,6 +184,8 @@ <h1 class="govuk-heading-xl">
             <input type="hidden" name="previous_url_name" value="
 
 
+
+
```
Comment (yusufsheiqh) [inline trade_remedies_public/templates/v2/feedback/collect_feedback.html:188]: A bunch of rogue lines here.

**Review by yusufsheiqh — APPROVED** [review]
Might be worth running `djhtml` on the two HTML files that have been edited in this PR, but otherwise, this looks good to me.

File: trade_remedies_public/templates/v2/feedback/collect_feedback.html (hunk 1 above)
Comment (chris-pettinga) [reply trade_remedies_public/templates/v2/feedback/collect_feedback.html:176]: unfortunately we're not running it as we're merging into master which doesn't have those changes yet

File: trade_remedies_public/templates/v2/feedback/collect_feedback.html (hunk 2 above)
Comment (chris-pettinga) [reply trade_remedies_public/templates/v2/feedback/collect_feedback.html:188]: good spot thanks, deleted

**Pull request merged**